src/net/RTP/RTPHeader.cs

```diff
@@ -78,26 +81,34 @@ public RTPHeader(byte[] packet)
                 SyncSource = BitConverter.ToUInt32(packet, 8);
             }
 
+
             Version = firstWord >> 14;
             PaddingFlag = (firstWord >> 13) & 0x1;
             HeaderExtensionFlag = (firstWord >> 12) & 0x1;
             CSRCCount = (firstWord >> 8) & 0xf;
+
             MarkerBit = (firstWord >> 7) & 0x1;
             PayloadType = firstWord & 0x7f;
 
+            int headerExtensionLength = 0;
+            // ReSharper disable once InconsistentNaming
             int headerAndCSRCLength = 12 + 4 * CSRCCount;

```

Comment:
Can these resharper warnings be done in a config file somewhere? It doesn't suit all of us to have these comments scattered throughout the code.

Reply:
sure,solved